lib/avahi.js

```diff
@@ -18,10 +18,12 @@ function supportsInterfaceIndexLocalOnly() {
           host, port, txtRec, cb, ctx);
   } catch (ex) {
     if (ex.errorCode === dns_sd.kDNSServiceErr_Unsupported) {
+      dns_sd.DNSServiceRefDeallocate(sr)
```

Comment:
Sure, no worries (a — not-so-undesirable — side-effect of working in CoffeeScript normally) :)